packages/lib/src/components/buttons/InfoButtonComponent.wc.svelte

```diff
@@ -3,18 +3,26 @@
         tag: "lens-info-button",
         props: {
             showQuery: { type: "Boolean" },
+            onlyChildInfo: { type: "Boolean" },
+            queryItem: { type: "Object" },
         },
     }}
 />
 
 <script lang="ts">
+    import { getHumanReadableQuery, buildHumanReadableRecursively } from "../../stores/negotiate";
+    import { returnNestedValues } from "../../helpers/ast-transformer";
+    import type { AstBottomLayerValue } from "../../types/ast";
+    import type { QueryItem } from "../../types/queryData";
     import { iconStore } from "../../stores/icons";
-    import { getHumanReadableQuery } from "../../stores/negotiate";
 
     export let message: string[] = [];
     export let noQueryMessage: string = "Search for all results";
     export let showQuery: boolean = false;
+    
     export let infoIconUrl: string | null = null;
+    export let onlyChildInfo: boolean = false;
+    export let queryItem: QueryItem;
```

Comment:
Here we should add `undefined` as a type and assign `undefined`, otherwise we get a warning, that it has an unused export variable, when it's not set.